release-notes.md

```diff
@@ -50,14 +20,55 @@ To test out the beta version for yourself,
 follow these [instructions](https://docs.timescale.com/clustering/getting-started/scaling-out).
 
 Currently, we expect this major feature to be released
-in the first half of 2020, and we will share more information once
+in the second half 2020, and we will share more information once
 it's available.
 
 ## Release Notes
 
 In this section, we will cover historical information on
 past releases and how you can learn more.
 
+
+### 1.7.0 (2020-04-16)
+
+This release adds major new features and bugfixes since the 1.6.1 release. We deem it moderate priority for upgrading.
+This release adds the long-awaited support for PostgreSQL 12 to TimescaleDB.
+This release also adds a new default behavior when querying continuous aggregates that we call real-time aggregation. A query on a 
+continuous aggregate will now combine materialized data with recent data that has yet to be materialized.
+
+Note that only newly created continuous aggregates will have this real-time query behavior, although it can be enabled on existing continuous aggregates 
+with a configuration setting as follows:
+
+ALTER VIEW continuous_view_name SET (timescaledb.materialized_only=false);
+
+This release also moves several data management lifecycle features to the Community version of TimescaleDB (from Enterprise), 
+including data reordering and data retention policies.
+
+**Major Features**
+*	#1456 Add support for PostgreSQL 12
```

Comment:
can you link #1807 here instead because thats the PR actually implementing it

Reply:
I have made that update